spec-main/api-safe-storage-spec.ts

```diff
@@ -0,0 +1,102 @@
+import * as cp from 'child_process';
+import * as path from 'path';
+import { safeStorage } from 'electron/main';
+import { expect } from 'chai';
+import { emittedOnce } from './events-helpers';
+import { ifdescribe } from './spec-helpers';
+import * as fs from 'fs';
+
+/* isEncryptionAvailable returns false in Linux when running CI due to a mocked dbus. This stops
+* Chrome from reaching the system's keyring or libsecret. When running the tests with config.store
+* set to basic-text, a nullptr is returned from chromium,  defaulting the available encryption to false.
+*
+* Because all encryption methods are gated by isEncryptionAvailable, the methods will never return the correct values
+* when run on CI and linux.
```

Comment:
This is a shame, it would be a nice followup to have the keyring stuff mocked out in dbus in a way that would allow the tests to run. But I think it's OK to ship without this.

Reply:
created a [follow-up issue](https://github.com/electron/electron/issues/30424) so this doesn't get lost :)